setup-env.sh

```diff
@@ -312,6 +312,7 @@ if [ -z "$RUN_IN_CONTAINER" ]; then
     -e RLWRAP_HOME=$(pwd) \
     -e RUN_IN_CONTAINER=true \
     -e MIGRATOR_ALLOW_STOP_PATH="${MIGRATOR_ALLOW_STOP_PATH:-false}" \
+    -e REGISTRY="${DAP_REGISTRY:-wandisco}" \
```

Comment:
we should use the same env name outside and inside to avoid misunderstandings how to change registry